src/api/package/assembly.ts

```diff
@@ -2,14 +2,22 @@ import * as spec from "@jsii/spec";
 import { API_PATHS } from "../../constants/url";
 import { getAssetsPath } from "./util";
 
+// These fields are removed from assembly.json during processing to save space,
+// and we don't need them here in the client.
+// See https://github.com/cdklabs/construct-hub/pull/567
+export type SlimAssembly = Omit<
+  spec.Assembly,
+  "types" | "readme" | "dependencyClosure"
```

Comment:
Where is the `readme` and `dependencyClosure` removed? I only see `types` being removed.

https://github.com/cdklabs/construct-hub/blob/main/src/backend/ingestion/ingestion.lambda.ts#L106

Reply:
Still haven't gotten around to it, but I am on it.

Reply:
https://github.com/cdklabs/construct-hub/pull/573